### PR TITLE
Devsprint 2023

### LINKED ISSUE #432: Add blog for PyCon India 2023 Devsprints
Something in similar lines to https://in.pycon.org/2019/devsprints.html and explaining the purpose of Devsprints

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
Closes #432

### CODE REVIEW

File: content/blog/2023/announcing-devsprints.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## When and Where?
+
+**After the conference, we will be organizing DevSprints on 3rd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
+
+The timings for the DevSprints will be from 10:00 AM to 5:00 PM.
+
+### How to participate?
+
+You do not need any additional tickets to participate in the DevSprints. If you have purchased a conference ticket, you can attend the DevSprints as well. However the capacity of the venue is limited, so we will be sharing a form to register for the DevSprints.
```
Comment (ananyo2012) [inline content/blog/2023/announcing-devsprints.md:29]: Any person with a Valid Conference ticket is free to attend Devsprints, subject to the maximum capacity of 200 participants. Entry will be permitted on a first come first serve basis at the Venue.

Comment (Sc0rpi0n101) [reply content/blog/2023/announcing-devsprints.md:29]: we're not sharing a form?

File: content/blog/2023/announcing-devsprints.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Who can participate?
+
+***You!*** Whether you're a seasoned contributor or just starting, the DevSprints at PyCon India 2023 are the perfect opportunity to get to work. Our mentors will be on hand to provide instant help whenever you need it, so you can keep moving forward. It is a great opportunity to get accustomed to the ways of the open-source and contribute to the projects you love.
+
+> `What good is a tech conference without getting your hands dirty writing some code?`
+
+## When and Where?
+
+**After the conference, we will be organizing DevSprints on 3rd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
```
[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply content/blog/2023/announcing-devsprints.md:23]:
```suggestion
**After the conference, we will be organizing DevSprints on 2nd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
```

File: content/blog/2023/announcing-devsprints.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The timings for the DevSprints will be from 10:00 AM to 5:00 PM.
```
[the remark this answers is not in the document]
Comment (Sc0rpi0n101) [reply content/blog/2023/announcing-devsprints.md:25]:
```suggestion
The timings for the DevSprints will be from 09:00 AM to 5:00 PM.
```

File: content/blog/2023/announcing-devsprints.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+If you're a contributor or maintainer of an open-source project, we invite you to mentor your project in the PyCon India 2023 DevSprints. Ideally, you should have a history with the project through previous code commits. You will help new contributors set up their development environment, list out good first issues that new contributors can easily get started with, and guide folks on any problems they face during the event.
+
+### How to add your Project?
+
+You can head over to [Devsprints CFP](https://in.pycon.org/cfp/devsprint-2023/proposals/) and submit your proposal.
+
+**Any maintainer is welcome to add a project. There will be no shortlisting of projects. We are using the CFP platform to make it easier to manage and keep the projects in one place.**
+
+If you have any questions regarding the dev sprints, be sure to reach out to us on Zulip over on the [`#2023/devsprint` stream!](https://pyconindia.zulipchat.com/)
```
Comment (ananyo2012) [inline content/blog/2023/announcing-devsprints.md:41]: Need to create this channel. Will provide channel link here.

File: content/blog/2023/announcing-devsprints.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2023/announcing-devsprints.md:29]:
```suggestion
You do not need any additional tickets to participate in the DevSprints. Any person with a Valid Conference ticket is free to attend Devsprints. However, the capacity of the venue is limited, up to a maximum of 200 participants. Entry will be permitted on a first come first serve basis at the Venue.
```

File: content/blog/2023/venue-announcement.md [hunk 5]
```diff
@@ -23,6 +23,10 @@ The venue is located in the Kukatpally area of Hyderabad. The nearest metro stat
 
 For more details, you can read our [Travel Guide](https://in.pycon.org/2023/travel/).
 
+#### Update: DevSprints Venue - Phenom People Pvt Ltd
+
+**After the conference, we will be organizing DevSprints on 3rd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
```
Comment (Sc0rpi0n101) [inline content/blog/2023/venue-announcement.md:28]:
```suggestion
**After the conference, we will be organizing DevSprints on 2nd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
```

File: content/blog/2023/announcing-devsprints.md (hunk 1 above)
Comment (ananyo2012) [reply content/blog/2023/announcing-devsprints.md]: Nope no form this time.

File: content/blog/2023/announcing-devsprints.md (hunk 4 above)
Comment (ananyo2012) [reply content/blog/2023/announcing-devsprints.md:41]: https://pyconindia.zulipchat.com/#narrow/stream/405903-2023.2Fdevsprints

**Review by ananyo2012 — APPROVED** [review]

File: content/blog/2023/announcing-devsprints.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+If you're a contributor or maintainer of an open-source project, we invite you to mentor your project in the PyCon India 2023 DevSprints. Ideally, you should have a history with the project through previous code commits. You will help new contributors set up their development environment, list out good first issues that new contributors can easily get started with, and guide folks on any problems they face during the event.
+
+### How to add your Project?
+
+You can head over to [Devsprints CFP](https://in.pycon.org/cfp/devsprint-2023/proposals/) and submit your proposal.
+
+**Any maintainer is welcome to add a project. There will be no shortlisting of projects. We are using the CFP platform to make it easier to manage and keep the projects in one place.**
+
+If you have any questions regarding the dev sprints, be sure to reach out to us on Zulip over on the [`#2023/devsprint` stream!](https://pyconindia.zulipchat.com/#narrow/stream/405903-2023.2Fdevsprints)
```
Comment (ananyo2012) [inline content/blog/2023/announcing-devsprints.md:41]: `#2023/devsprints`

Comment (Sc0rpi0n101) [reply content/blog/2023/announcing-devsprints.md:41]:
```suggestion
If you have any questions regarding the dev sprints, be sure to reach out to us on Zulip over on the [`#2023/devsprints` stream!](https://pyconindia.zulipchat.com/#narrow/stream/405903-2023.2Fdevsprints)
```

**Pull request merged**